css/style.css

```diff
@@ -0,0 +1,442 @@
+	margin:4rem 0 2rem 0;
+	border-top:1px solid #e5e5e5;
+	padding:2rem 0;
+}
+
+.cat-title{
+	color: #f72210;
+	font-weight: bold;
+}
+
+.category-box .third-col .post{
+	padding-top:.7rem;
+	border-bottom:1px solid #e5e5e5;
+	padding-bottom:.7rem;
+}
+
+.nw-img{
+	height: 100%;
+	object-fit: cover;
+}
+
+.newsletter-row{
+	border:1px solid #e5e5e5;
+	max-width: 98%;
+	margin:auto;
+}
+
+.search-bar{
+	border:none;
+	background: transparent;
+	font-size: 1rem;
+}
+
+.fa-search{
+	position: absolute;
+	left: 50%;
+}
+
+.p-relative{
+	position: relative;
+}
+
+.highlights{
+	color:#666;
+}
+
+.post h4{
+  font-weight: bold;
+}
+
+
+.post h4, .post p{
+	color:#000;
+
+}
+
+.features{
+	background: #f72210;
+	color: #fff;
+	font-weight: bold;
+	padding:.3rem .8rem;
+	display: inline-block;
+}
+
+.p-category{
+	color:#f72210;
+	font-weight: bold;
+}
+
+
+
+.post h4:hover,.post h5:hover,.post h6:hover{
+	text-decoration: underline;
+	text-decoration-color: #f72210;
+	color:#000;
+}
+
+.post a{
+	text-decoration: none;
+}
+
+.post-img-frame{
+	position: relative;
+}
+
+.post-category{
+	position: absolute;
+	left:0;
+	background: rgba(255,255,255,0.75);
+	color:#f72210;
+	padding:.1rem .5rem;
+	font-weight: bold;
+	bottom:0;
+}
+
+.ads-category{
+	color:steelblue;
+}
+
+.ad-box{
+	border:1px solid #e5e5e5;
+	padding:1rem 1.5rem;
+	margin-bottom: 1rem;
+}
+
+.featured-stories-col{
+	padding:0rem 1.5rem;
+}
+
+.post-title{
+	font-weight: bold;
+}
+
+.weekly-mid-sec,.subscribe-row{
+	background:#F4F5F6;
+}
+
+.subscribe-box{
+	background: #F4F5F6;
+	border:1px solid #e5e5e5;
+	padding:1.5rem;
+}
+
+
+
+.subscribe-sec .story-title{
+	font-weight: bold;
+	margin-top: 1.4rem;
+}
+
+.subscribe-sec {
+	border-bottom: 1px solid #e5e5e5;
+	padding-bottom: 2rem;
+}
+
+
+
+.social li{
+	list-style: none;
+}
+
+.social li a{
+	font-size: 1.3rem;
+	padding-right: 1rem;
+	color:#666;
+}
+
+.footer-navigation li{
+	list-style: none;
+	float: left;
+}
+
+.footer-navigation li a{
+	padding: 0 .8rem;
+	border-right: 1px solid #e5e5e5;
+	color:#666;
+	font-family: sans-serif;
+	font-size: .9rem;
+}
+
+.copyright-row{
+	max-width: 95%;
+	margin:0 auto 1rem auto;
+	border-bottom: 1px solid #e5e5e5;
+}
+
+
+.footer-logo{
+	max-width: 220px;
+}
+
+.btn-red{
+	color: #fff;
+	border-radius: 0;
+	padding:.5rem 1.2rem;
+	font-weight: bold;
+	background-color:#f72210;
+}
+
+.subscribe-row{
+	position: relative;
+	padding:2rem 0 4rem 0;
+	max-width: 95%;
+	margin:auto;
+}
+
+.s-img-one,.s-img-two{
+   position: absolute;
+   top:0;
+}
+
+.s-img-two{
+	left: 25%;
+	top:10%;
+}
+
+.trans-btn{
+	background: transparent;
+	color:#ddd;
+	display: inline;
+}
+
+
+.p-title{
+	font-weight: 300 !important;
+}
+
+@media(min-width:1200px){
+	.features-row img{
+	min-height: 300px;
+	object-fit: cover;
+  }
+}
+
+
+
+@media screen and (min-width: 992px){
+	.navbar::before{
+	    width: 66px;
+	    position: absolute;
+	    top: 0;
+	    right: 0;
+	    content: "";
+	    display: block;
```

Comment:
`SUGGESTION`

Consider using a Bootstrap class name for this selector element and in all other occurrences.